### DISCUSSION TITLE
"Delete files older than" - wrong order?

### DISCUSSION THREAD

**Discussion by chrysenteron** [opengisch/QField#3933] [Bugs]
Hi,
When deleting an old file version, there's a checkbox for deleting all older file versions with the text "Also delete X version(s) older than the selected version." 
However, X actually seems to be the number of newer versions.

Edit: I also get Server Error when the box is checked.

**Comment by ZsanettMed** [comment]
Hi, 
Thank you very much for reporting, we could reproduce the issue and will fix it as soon as possible, meanwhile you can delete older versions one by one. We apologize for the inconvenience!
Best regards,
Zsanett

**Reply by ZsanettMed** [reply]
merged with #3926

**Comment by ZsanettMed** [comment]
Hi @chrysenteron,
The bug has been fixed. Thank you again for reporting. Enjoy QFieldCloud! 
Kind regards, 
Zsanett
